Sources/SnowManLogic/GameLogic/Game.swift

```diff
@@ -0,0 +1,45 @@
+public class Game {
+	var word: String
+	var guessManager: GuessManager
+	var view: View
+	var guess: String
+
+	public init(word: String, guessManager: GuessManager, view: View) {
+		self.word = word
+		self.guessManager = guessManager
+		self.view = view
+		guess = ""
+	}
+
+	public func playGame() {
+		while (gameIsOver() == false) {
+			playerTurn()
+		}
+		view.displayResults(word, winner: isWinner())
+	}
+
+	public func gameIsOver() -> Bool {
+		return guessManager.hasNoGuessesRemaining() || isWinner()
+	}
+
+	public func isWinner() -> Bool {
+		let letters = separateLetters()
+		return guessManager.determineUnguessedLetters(letters).isEmpty || guessManager.correctlyGuessedFullWord(word, guess: guess)
+	}
+
+	private func playerTurn() {
+		view.assignBlanks(word, correctGuesses: guessManager.correctGuesses)
+		guess = view.receiveGuess(view.getGuessFromUser)
+		
+		if (guessManager.isGuessingFullWord(guess)) {
+			guessManager.appendGuess(word, guess: "#")
```

Comment:
Spoke with Malcolm about this. In order to keep track of `remainingGuesses`, I am subtracting the count of `incorrectGuesses` from the allowed number of guesses. 

Each time an incorrect letter guess is made towards the word, that guess is added to the `incorrectGuesses` array, so there would be one fewer remaining guess. 

In order to also decrement the remaining guesses when someone incorrectly guesses a word, I am passing this in to add to the count (as I cannot pass a `String` into a `[Character]` Array).

To make this more clear, I am assigning "#" to a constant called `incorrectFullWordGuessPlaceholder` and passing that in instead.